### PR TITLE
Add settings page (only appearance settings for now)

### PR DESCRIPTION
**Pull request by unbiaseduser**
I added:
- Light/dark/follow system mode, since every app has that option nowadays.
- This one is a bit unconventional: **_optional_** Material You theme! Yes, unlike other apps that only have either MD2 or MD3 theme, this app's gonna have both! Seriously, it's really easy to implement, I wonder why no one has ever done it. Having MD2 as an option means that ppl stuck with Android 11 or lower (like _cough cough_ me) won't be forced to put up with a radically different style from the rest of the system, while still be able to get a glimpse of future versions of Android if they feel like it. 

Tl;dr:  Having more options is always better for everyone.
<h6>also i wont pull on master anymore im sorry</h6>

### CODE REVIEW

**Comment by KiARC** [comment]
That's all so cool!

**Comment by KiARC** [comment]
Alright I finally have some time to look over this, sorry for taking so long. End of quarter stuff was a mess.
